README_FA.md

```diff
@@ -0,0 +1,21 @@
+<div dir="rtl">
+
+# پیوستن به تیم ما
+
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-7-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+[![README.md](https://img.shields.io/badge/English-up-brightgreen)](README.md)
+[![README_JO.md](https://img.shields.io/badge/Arabic-up-brightgreen)](README_JO.md)
```

Comment:
Can you  please add a button here for Farsi ?

Reply:
Sure, allow me a few moments.